molo/core/api/tests/utils.py

```diff
@@ -0,0 +1,59 @@
+from .constants import (
+    AVAILABLE_ARTICLES, AVAILABLE_SECTIONS,
+    AVAILABLE_SECTION_CHILDREN, RELATED_IMAGE
+)
+
+
+# Inspired by http://stackoverflow.com/a/28507806
```

Comment:
Would be easier to use `requests` and [responses](https://github.com/getsentry/responses) for handling HTTP requests in tests